docs/using/handlers.md

```diff
@@ -0,0 +1,164 @@
+abstract chemical environments like SMIRNOFF force fields, or copying
+parameters into place like a traditional force field format, a
+`PotentialHandler` links an input parameter in the force field to every place
+in the topology where it is used.
+
+Like SMIRNOFF force fields, each parameter in an `Interchange`  knows where it
+came from, but like traditional force fields, the parameterized system is
+readily defined and values can be retrieved quickly. This allows changes to an
+input parameter to be reflected instantly in the parameterized system. Unlike a
+SMIRNOFF force field, the chemistry of system itself cannot be changed; a new
+`Interchange` must be defined and parameterized.
+
+There are three central components in each handler: topology keys, potential keys,
+and potentials.
+
+[`TopologyKey`] objects are unique identifiers of locations in a topology. These
+objects do not include physics parameters. The basic information is a tuple of
+atom indices, which can be of any non-zero length. For example, a topology key
+describing a torsion will have a 4-length tuple, and a topology key describing
+a vdW parameter will have a 1-length tuple.
+
+[`PotentialKey`] objects are unique identifiers of physics parameters. These
+objects do not know anything about the topology they are associated with. In
+SMIRNOFF force fields, SMIRKS patterns uniquely identify a parameter within a
+parameter handler, so (with some exceptions) that is all that is needed to
+construct a potential key. For classically atom-typed force fields, a key can
+be constructed using atom types or combinations thereof.
+
+[`Potential`] objects store the physics parameters that result from parameterizing
+a chemical topology with a force field. These do not know anything about where
+in the topology they are associated. The parameters are stored in a dictionary
+attribute `.parameters` in which keys are string identifiers and values are the
+parameters themselves, tagged with units.
+
+These objects are strung together with two mappings, each stored as dictionary
+attributes of a `PotentialHandler`. The `.slot_map` attribute maps segments of
+a topology to the potential keys (`TopologyKey` to `PotentialKey` mapping). The
+`.potentials` attribute maps the potential keys to the potentials
+(`PotentialKey` to `Potential`). This allows many topology keys to map to the
+same `Potential` by sharing a `PotentialKey`. If the `Potential` is updated,
+all the places in the topology where it is used are updated immediately.
+Despite this, getting the `Potential` for a place in the topology is a constant
+time operation.
+
+Each potential handler inherits from the base [`PotentialHandler`] class and
+describes a single type of parameter from a single source. Potential handlers
+for SMIRNOFF force fields are found in the [](openff.interchange.components.smirnoff)
+module, while those for Foyer are found in the [](openff.interchange.components.foyer).
+module.
+
```

Comment:
This doesn't render for me; not an issue if it looks good in the RTD builds.

![image](https://user-images.githubusercontent.com/7935382/161120967-f6d9e873-cd11-4498-82e4-23f82ec37e33.png)

Reply:
Whoops these don't even work for me locally.

Reply:
Fixed